pkg/awsclient/client.go

```diff
@@ -312,3 +315,36 @@ func NewClient(kubeClient client.Client, secretName, namespace, region string) (
 		tagClient:     resourcegroupstaggingapi.New(s),
 	}, nil
 }
+
+// NewClientFromEnvVars creates our client wrapper object for the actual AWS clients we use.
+// For authentication the underlying clients will use the standard AWS credentials env
+// vars.
+func NewClientFromEnvVars(region string) (Client, error) {
```

Comment:
Can this function simply call `NewClient(nil, "", "", region)`?

Note that `session.NewSession` will read the environment variables for the access key.